--- yap-6.2.2/Makefile.in.orig	2011-10-20 23:17:58.000000000 +0200
+++ yap-6.2.2/Makefile.in	2012-03-28 19:44:30.763163899 +0200
@@ -9,7 +9,7 @@
 #
 # where the binary should be
 #
-BINDIR = $(EROOTDIR)/bin
+BINDIR = @bindir@
 #
 # where YAP should look for binary libraries
 #
@@ -18,15 +18,15 @@
 #
 # where YAP should look for architecture-independent Prolog libraries
 #
-SHAREDIR=$(ROOTDIR)/share
+SHAREDIR=@datadir@
 #
 # where the includes should be stored
 #
-INCLUDEDIR=$(ROOTDIR)/include/Yap
+INCLUDEDIR=@includedir@/Yap
 #
 # where to store info files
 #
-INFODIR=$(SHAREDIR)/info
+INFODIR=@infodir@
 #
 # where to store documentation files
 #
--- yap-6.2.2/library/Makefile.in.orig	2011-02-03 12:25:28.000000000 +0100
+++ yap-6.2.2/library/Makefile.in	2012-03-28 19:49:09.476500101 +0200
@@ -5,7 +5,7 @@
 #
 # where the binary should be
 #
-BINDIR = $(ROOTDIR)/bin
+BINDIR = @bindir@
 #
 # where YAP should look for binary libraries
 #
@@ -13,7 +13,7 @@
 #
 # where YAP should look for architecture-independent Prolog libraries
 #
-SHAREDIR=$(ROOTDIR)/share
+SHAREDIR=@datadir@
 #
 #
 # You shouldn't need to change what follows.
